Dockerfile-build

```diff
@@ -0,0 +1,77 @@
+# This is an image that should been able to generate Swift libsbp bindings.
+#
+# Developers should be in the top level directory of a clean clone of
+# libsbp, then type:
+#
+# docker run -v `pwd`:/mnt/workspace -i -t <this image>
+#
+# where <this image> is
+# 571934480752.dkr.ecr.us-west-2.amazonaws.com/libsbp-build:<tag>
```

Comment:
These instructions look slightly different than the ones in the readme - can we have only one set?

Reply:
Was testing the instructions and I think @schniedergers needs to add a new entry to ECR first so I can publish this.

Reply:
You can add it here: https://github.com/swift-nav/cloud-infra/blob/master/terraform/ci/ecr/ci_ecr.tf

Reply:
I am looking at the comment, and I am confused about what I am supposed to do:
```
// To import an existing ECR repo, log into the AWS build account,
// then run 'terraform import 'aws_ecr_repository.build-ecr["hello-swift"]' hello-swift' as example for
// the hello-swift repo.
//
// If you need to remove state for a repo, use 'terraform state rm aws_ecr_repository.build-ecr["hello-swift"]'.
```
I don't think there is a ECR repo for libsbp-build yet;  also, I don't have authorization to assume the build role, and this is not for CI, per se.  This is for devs so they don't have to create the docker image from scratch (which takes a while).

I just want to create the libsbp-build repo in ECR so tag pushes under `docker-recipes` for libsbp-build will generate images that we can access.

Reply:
How about we push this to our public repo then?  https://hub.docker.com/u/swiftnav  The `docker-recipes` repo has support for this (see the `buildroot-base` docker image).  This was we don't need to deal with ECR at all.

Reply:
See https://github.com/swift-nav/docker-recipes/blob/master/Jenkinsfile#L121 -- particularly the `dockerhub` parameter should be `true` and the `ecr` param `false`:
```
                        imageBuild(
                            context: context,
                            name: "buildroot-base",
                            dockerfile: "buildroot-base/piksi_buildroot/scripts/Dockerfile.base",
                            dockerhub: true,
                            ecr: false,
                            tags: tagInfo.imageTags,
                            variants: tagInfo.publishVariants,
                        )
```